src/components/storage-browser.less

```diff
@@ -68,6 +68,9 @@
 
 .StorageBrowser-tableDeleteCell {
```

Comment:
This class is not used anymore

Reply:
The class that started the extraction of the delete button, of course 🤦‍♀️ 
Removed, thank you!